frontend/front/src/util/stringUtils.js

```diff
@@ -1,7 +1,13 @@
 // given a sentence, get the first X words, lowercased and hyphenated
-export const sentenceAsLink = (string) =>
-  string
+export const sentenceAsLink = (string) => {
+  return string
     .split(" ")
     .slice(0, 10)
     .map((s) => s.replace(/[^\w\s]/g, "").toLowerCase())
     .join("-");
+};
+
+// Mostly used to prepend 0xxxx zipcodes with a 0 instead of displaying as a 4 digit number.
+export const formatZipcode = (zipCode) => {
+  return zipCode.length === 5 ? zipCode : `0${zipCode}`;
```

Comment:
Can confirm they're stored as strings on the backend – however I think due to an editor error, when I created the dummy data they might briefly have been turned to numbers and lost their leading zeros.

Reply:
Ah, that makes sense then. I think it might be best then to take this out and assume that the production build will just send strings with leading zeroes and all.